src/Slick/Caching.hs

```diff
@@ -63,10 +64,10 @@ jsonCache' loader = unpackJSON
 
 -- | A wrapper around 'jsonCache' which simplifies caching of values which do NOT
 -- depend on an input parameter. Unfortunately Shake still requires that the
--- key type implement several typeclasses, however this is easily accomplished 
+-- key type implement several typeclasses, however this is easily accomplished
```

Comment:
It looks to me like your editor adds a lot of irrelevant spaces in random places.
Could you manage to somehow remove these changes from diff?

Reply:
Yes, I'll try to revert them. I use regular Emacs. It's actually deletes spaces aggressively, not sure what's wrong with comments.